modules/ckanext-ytp_main/ckanext/ytp/plugin.py

```diff
@@ -264,16 +263,16 @@ def dataset_facets(self, facets_dict, package_type):
         lang = get_lang_prefix()
         facets_dict = OrderedDict()
         facets_dict.update({'vocab_international_benchmarks': _('International Benchmarks')})
+        facets_dict.update({'vocab_geographical_coverage': _('Geographical coverage')})
         facets_dict.update({'collection_type': _('Collection Types')})
-        facets_dict['vocab_keywords_' + lang] = _('Popular Tags')
-        facets_dict.update({'vocab_content_type_' + lang: _('Content Types')})
-        facets_dict.update({'organization': _('Organizations')})
+        facets_dict['vocab_keywords_' + lang] = _('Tags')
+        facets_dict.update({'organization': _('Organization')})
         facets_dict.update({'res_format': _('Formats')})
-        # BFW: source is not part of the schema. created artificially at before_index function
-        facets_dict.update({'source': _('Sources')})
         facets_dict.update({'license_id': _('Licenses')})
-        # add more dataset facets here
+        facets_dict.update({'groups': _('Category')})
         facets_dict.update({'producer_type': _('Producer type')})
+        # BFW: source is not part of the schema. created artificially at before_index function
```

Comment:
This comment could be removed as it refers to facet that no longer exists.

Reply:
Fixed in 324de87e3974505ecb310d562bffd1ebc486ae24